### ISSUE TITLE
go get fails

### ISSUE THREAD

**Issue by ijt** [Azure/azure-event-hubs-go#63]
### Expected Behavior
```
$ go get -u github.com/Azure/azure-event-hubs-go/...
```
should run without failing.

### Actual Behavior
It fails with this error message:
```
# github.com/Azure/azure-amqp-common-go/aad
../../src/github.com/Azure/azure-amqp-common-go/aad/jwt.go:203:44: cannot use token.ExpiresOn (type json.Number) as type string in argument to strconv.ParseInt
../../src/github.com/Azure/azure-amqp-common-go/aad/jwt.go:216:69: cannot use token.ExpiresOn (type json.Number) as type string in argument to auth.NewToken
```

### Environment
- OS: Debian GNU/Linux
- Go version: go1.11rc2 linux/amd64
- Version of Library: ed70919012a66645f619384dced1f8cb3d2d7a44

**Comment by ijt** [comment]
Hmm, I guess I filed this on the wrong repo. Looks like this issue belongs on azure-amqp-common-go.

**Comment by slamgundam** [comment]
i'm also facing same issue.
Have any solution on this error?

**Comment by devigned** [comment]
@slamgundam are you seeing the same error as above? This should have been resolved via: https://github.com/Azure/azure-amqp-common-go/issues/26 and https://github.com/Azure/azure-amqp-common-go/issues/27.

**Comment by slamgundam** [comment]
> @slamgundam are you seeing the same error as above? This should have been resolved via: [Azure/azure-amqp-common-go#26](https://github.com/Azure/azure-amqp-common-go/issues/26) and [Azure/azure-amqp-common-go#27](https://github.com/Azure/azure-amqp-common-go/pull/27).

yup same error.  I just simply copy there sample and only change the connectionString, that's all

**Comment by slamgundam** [comment]
@devigned 

Thanks! Just got it work!
Guess i had an old repo :)
